Build failure: openusd

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

From a checkout of `nixpkgs` `master`,
```console
$ nix-build -A python311Packages.openusd
```

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/288117448

### Relevant log output

```console
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/gqjblggiim5dm1n2sjnhkka22fwaybz4-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file "source/third_party/renderman-26/shaders/UsdPreviewSurfaceParameters.osl"
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
applying patch /nix/store/gp6as7g881cpki68kf2q942ccyacy1jq-port-to-embree-4.patch
patching file cmake/modules/FindEmbree.cmake
patching file pxr/imaging/plugin/hdEmbree/context.h
patching file pxr/imaging/plugin/hdEmbree/mesh.cpp
patching file pxr/imaging/plugin/hdEmbree/mesh.h
patching file pxr/imaging/plugin/hdEmbree/meshSamplers.h
patching file pxr/imaging/plugin/hdEmbree/pch.h
patching file pxr/imaging/plugin/hdEmbree/renderDelegate.h
patching file pxr/imaging/plugin/hdEmbree/renderParam.h
patching file pxr/imaging/plugin/hdEmbree/renderer.cpp
patching file pxr/imaging/plugin/hdEmbree/renderer.h
patching file pxr/imaging/plugin/hdEmbree/testenv/testHdEmbree.cpp
applying patch /nix/store/rb1rpl57is44vznhvhwhf4xmsgc1nkaa-explicitly-adding-template-keyword.patch
patching file pxr/usd/sdf/children.cpp
applying patch /nix/store/bvwvz08blf2i6zp2pd3wvpyxd7289z5x-fix-removes-unused-path.patch
patching file pxr/usd/sdf/childrenProxy.h
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -GNinja -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/z8hbp04qjz26a9xrm9xl1d9vsjajglgc-python3.11-openusd-24.11/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/z8hbp04qjz26a9xrm9xl1d9vsjajglgc-python3.11-openusd-24.11/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/z8hbp04qjz26a9xrm9xl1d9vsjajglgc-python3.11-openusd-24.11/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/z8hbp04qjz26a9xrm9xl1d9vsjajglgc-python3.11-openusd-24.11/share/doc/usd -DCMAKE_INSTALL_INFODIR=/nix/store/z8hbp04qjz26a9xrm9xl1d9vsjajglgc-python3.11-openusd-24.11/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/z8hbp04qjz26a9xrm9xl1d9vsjajglgc-python3.11-openusd-24.11/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/z8hbp04qjz26a9xrm9xl1d9vsjajglgc-python3.11-openusd-24.11/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/z8hbp04qjz26a9xrm9xl1d9vsjajglgc-python3.11-openusd-24.11/include -DCMAKE_INSTALL_SBINDIR=/nix/store/z8hbp04qjz26a9xrm9xl1d9vsjajglgc-python3.11-openusd-24.11/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/z8hbp04qjz26a9xrm9xl1d9vsjajglgc-python3.11-openusd-24.11/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/z8hbp04qjz26a9xrm9xl1d9vsjajglgc-python3.11-openusd-24.11/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/4ijy8jbsiqmj37avrk83gn2m903486mr-gcc-wrapper-14-20241116/bin/strip -DCMAKE_RANLIB=/nix/store/4ijy8jbsiqmj37avrk83gn2m903486mr-gcc-wrapper-14-20241116/bin/ranlib -DCMAKE_AR=/nix/store/4ijy8jbsiqmj37avrk83gn2m903486mr-gcc-wrapper-14-20241116/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/z8hbp04qjz26a9xrm9xl1d9vsjajglgc-python3.11-openusd-24.11 -DPXR_BUILD_ALEMBIC_PLUGIN=ON -DPXR_BUILD_DRACO_PLUGIN=ON -DPXR_BUILD_EMBREE_PLUGIN=ON -DPXR_BUILD_EXAMPLES=OFF -DPXR_BUILD_IMAGING=ON -DPXR_BUILD_MONOLITHIC=ON -DPXR_BUILD_TESTS=OFF -DPXR_BUILD_TUTORIALS=OFF -DPXR_BUILD_USD_IMAGING=ON -DPYSIDE_BIN_DIR=/nix/store/cyxa58gjjy8xg26ypx91v96alahm7mx3-pyside6-uic/bin -DPXR_BUILD_DOCUMENTATION:BOOL=FALSE -DPXR_BUILD_PYTHON_DOCUMENTATION:BOOL=FALSE -DPXR_BUILD_USDVIEW:BOOL=FALSE -DPXR_BUILD_USD_TOOLS:BOOL=FALSE -DPXR_ENABLE_MATERIALX_SUPPORT:BOOL=TRUE -DPXR_ENABLE_OSL_SUPPORT:BOOL=TRUE
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/4ijy8jbsiqmj37avrk83gn2m903486mr-gcc-wrapper-14-20241116/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/4ijy8jbsiqmj37avrk83gn2m903486mr-gcc-wrapper-14-20241116/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found Python3: /nix/store/gzl581vkwymk2iys5zz6k1k1zdf03ba1-python3-3.11.11/bin/python3.11 (found version "3.11.11") found components: Interpreter Development Development.Module Development.Embed
-- Found Jinja2
-- Found TBB: /nix/store/vbnp9cdf34ahixgr5ni4j2s9fhlcl7wx-tbb-2020.3-dev/include (found version "2020.3") found components: tbb
-- Using default system allocator because PXR_MALLOC_LIBRARY is unspecified
CMake Deprecation Warning at cmake/defaults/Packages.cmake:208 (cmake_policy):
  The OLD behavior for policy CMP0072 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.
Call Stack (most recent call first):
  CMakeLists.txt:23 (include)


-- Found OpenGL: /nix/store/8a6na7jyfg3dr9z7pksijpp832632j98-libglvnd-1.7.0/lib/libOpenGL.so
-- Found OpenSubdiv: /nix/store/pkp3zbp5msr283y3pgqjqc8k9km2dd9l-opensubdiv-3.6.0-dev/include (found suitable version "3.6.0", minimum required is "3")
-- Found X11: /nix/store/64ia61iphq1ciqlvgra9d9rzfgycfy5k-xorgproto-2024.1/include
-- Looking for XOpenDisplay in /nix/store/80r6vnyb23f84b4jx3qdb9spy3f02akk-libX11-1.8.10/lib/libX11.so
-- Looking for XOpenDisplay in /nix/store/80r6vnyb23f84b4jx3qdb9spy3f02akk-libX11-1.8.10/lib/libX11.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Found Embree: /nix/store/hciwhw4v02nj45d6x515rl3y81m3h9vh-embree-4.3.3/include
-- Found Alembic: /nix/store/49qv3r568bjf226kf3h66vwkjwzw8iai-alembic-1.8.8-lib/lib/libAlembic.so
-- Found OSL: /nix/store/83wviydzjvd6p74vlw55z852aixd2c3d-openshadinglanguage-1.13.11.0/include
-- C++ namespace configured to (external) pxr, (internal) pxrInternal_v0_24_11
-- Skipping validation of gf generated code because PXR_VALIDATE_GENERATED_CODE=OFF
-- Skipping commandline tools because PXR_BUILD_USD_TOOLS=OFF
-- Skipping hgiMetal because PXR_BUILD_GPU_SUPPORT or PXR_ENABLE_METAL_SUPPORT is OFF
-- Skipping hgiVulkan because PXR_BUILD_GPU_SUPPORT or PXR_ENABLE_VULKAN_SUPPORT is OFF
-- Configuring done (3.7s)
CMake Error at /nix/store/6narii25bad229b8ybvsr2dqj17gx84q-python3.11-materialx-1.39.1/lib/cmake/MaterialX/MaterialXTargets.cmake:222 (set_target_properties):
  The link interface of target "MaterialXRender" contains:

    OpenImageIO::OpenImageIO

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  /nix/store/6narii25bad229b8ybvsr2dqj17gx84q-python3.11-materialx-1.39.1/lib/cmake/MaterialX/MaterialXConfig.cmake:36 (include)
  cmake/defaults/Packages.cmake:306 (find_package)
  CMakeLists.txt:23 (include)


-- Generating done (1.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_INSTALL_BINDIR
    CMAKE_INSTALL_DOCDIR
    CMAKE_INSTALL_INCLUDEDIR
    CMAKE_INSTALL_INFODIR
    CMAKE_INSTALL_LIBDIR
    CMAKE_INSTALL_LIBEXECDIR
    CMAKE_INSTALL_LOCALEDIR
    CMAKE_INSTALL_MANDIR
    CMAKE_INSTALL_OLDINCLUDEDIR
    CMAKE_INSTALL_SBINDIR
    CMAKE_POLICY_DEFAULT_CMP0025
    PYSIDE_BIN_DIR


CMake Generate step failed.  Build files cannot be regenerated correctly.
```

### Additional context

Discovered while trying to build `blender`.

### System metadata

- system: `"x86_64-linux"`
- host os: `Linux 6.12.11, NixOS, 25.05 (Warbler), 25.05.20250126.852ff1d`
- multi-user?: `yes`
- sandbox: `yes`
- version: `nix-env (Lix, like Nix) 2.92.0
System type: x86_64-linux
Additional system types: i686-linux, x86_64-v1-linux, x86_64-v2-linux, x86_64-v3-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /home/aidan/.config/nix/nix.conf:/home/aidan/.config/kdedefaults/nix/nix.conf:/nix/store/sw9amjplk1al9ik84ln9scqqsx32x3dl-plasma-workspace-6.2.5/etc/xdg/nix/nix.conf:/nix/store/lsdmjyz51k5405d7im89yw68wqp4dymd-kglobalacceld-6.2.5/etc/xdg/nix/nix.conf:/nix/store/xw625nr26j3lfxch9ciw0c7gcxsj7bq7-baloo-6.10.0/etc/xdg/nix/nix.conf:/nix/store/sw9amjplk1al9ik84ln9scqqsx32x3dl-plasma-workspace-6.2.5/etc/xdg/nix/nix.conf:/nix/store/lsdmjyz51k5405d7im89yw68wqp4dymd-kglobalacceld-6.2.5/etc/xdg/nix/nix.conf:/nix/store/xw625nr26j3lfxch9ciw0c7gcxsj7bq7-baloo-6.10.0/etc/xdg/nix/nix.conf:/home/aidan/.config/kdedefaults/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/aidan/.nix-profile/etc/xdg/nix/nix.conf:/nix/profile/etc/xdg/nix/nix.conf:/home/aidan/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/aidan/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/jzxdqpm0xjsbr8k0rj7drlzpybmay8pj-lix-2.92.0/share`
- nixpkgs: `/home/aidan/.nix-defexpr/channels/nixpkgs`

### Notify maintainers

@ShaddyDC @gador 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/materialx/default.nix
+++ b/pkgs/development/python-modules/materialx/default.nix
@@ -64,6 +64,12 @@ buildPythonPackage rec {
     ln -s $out/python $target_dir
   '';
 
+  # Update to 1.39 has major API changes and downstream software
+  # needs to adapt, first. So, do not include in mass updates. For reference, see
+  # https://github.com/NixOS/nixpkgs/pull/326466#issuecomment-2293029160
+  # and https://github.com/NixOS/nixpkgs/issues/380230
+  passthru.skipBulkUpdate = true;
+
   meta = {
     changelog = "https://github.com/AcademySoftwareFoundation/MaterialX/blob/${src.rev}/CHANGELOG.md";
     description = "Open standard for representing rich material and look-development content in computer graphics";

--- a/pkgs/development/python-modules/materialx/default.nix
+++ b/pkgs/development/python-modules/materialx/default.nix
@@ -16,13 +16,13 @@
 
 buildPythonPackage rec {
   pname = "materialx";
-  version = "1.39.1";
+  version = "1.38.10";
 
   src = fetchFromGitHub {
     owner = "AcademySoftwareFoundation";
     repo = "MaterialX";
-    tag = "v${version}";
-    hash = "sha256-WzzsY1hOWwJEqT/ZRLIoZDfKNvx1Yf6aFhA3ZcSPx+s=";
+    rev = "v${version}";
+    hash = "sha256-/kMHmW2dptZNtjuhE5s+jvPRIdtY+FRiVtMU+tiBgQo=";
   };
 
   format = "other";
@@ -65,7 +65,7 @@ buildPythonPackage rec {
   '';
 
   meta = {
-    changelog = "https://github.com/AcademySoftwareFoundation/MaterialX/blob/${src.tag}/CHANGELOG.md";
+    changelog = "https://github.com/AcademySoftwareFoundation/MaterialX/blob/${src.rev}/CHANGELOG.md";
     description = "Open standard for representing rich material and look-development content in computer graphics";
     homepage = "https://materialx.org";
     maintainers = [ lib.maintainers.gador ];

